packages/react/src/lib/whatInput.ts

```diff
@@ -212,17 +214,55 @@ export const setUpWhatInput = (target: Document) => {
     'addEventListener' in targetWindow &&
     Array.prototype.indexOf
   ) {
-    const whatInputInitialized = 'whatInputInitialized'
-    if (target[whatInputInitialized] === true) {
+    if (target[whatInputInitialized]) {
+      target[whatInputInitialized] = target[whatInputInitialized] + 1
       return
     }
-    target[whatInputInitialized] = true
+    target[whatInputInitialized] = 1
 
     addListeners(targetWindow)
     doUpdate(target)
   }
 }
 
+function cleanupWhatInput(eventTarget: Window) {
+  const options = supportsPassive ? { capture: true } : true
+
+  // @ts-ignore
+  if (eventTarget.PointerEvent) {
+    eventTarget.removeEventListener('pointerdown', setInput)
+    // @ts-ignore
+  } else if (window.MSPointerEvent) {
```

Comment:
still thinking why here there is `window` and not `eventTarget`.

Reply:
I am replacing this. @jurokapsiar @layershifter any comments?

Reply:
this is basically a platform check - saying "does this browser have MSPointerEvent?"

Reply:
will revert it then, but shouldn't the check above mean similarly? `eventTarget.PointerEvent`